src/models/room-state.js

```diff
@@ -68,9 +68,7 @@ export function RoomState(roomId, oobMemberFlags = undefined) {
     this.members = {
         // userId: RoomMember
     };
-    this.events = {
-        // eventType: { stateKey: MatrixEvent }
-    };
+    this.events = new Map(); // { eventType: { stateKey: MatrixEvent } }
```

Comment:
Can we update this comment to mention that the outer map values are also maps and not dicts